tests/common/helpers/dut_utils.py

```diff
@@ -230,3 +230,33 @@ def decode_dut_and_container_name(name_str):
         container_name = name_list[0]
 
     return dut_name, container_name
+
+
+def verify_features_state(duthost):
+    """Checks whether the state of each feature is valid.
+
+    Args:
+      duthost: An Ansible object of DuT.
+
+    Returns:
+      None.
+    """
+    feature_status, succeeded = duthost.get_feature_status()
+    if not succeeded:
+        logger.info("Failed to get list of feature names.")
+        return False
+
+    for feature_name, status in feature_status.items():
+        get_command = 'sonic-db-cli CONFIG_DB HGET "FEATURE|{}" state'.format(feature_name)
+        get_command_result = duthost.shell(get_command)["stdout_lines"][0]
+        feature_state = get_command_result.strip()
```

Comment:
Why getting state again when you already have it in status? Why not just check status?

Reply:
I should reuse the `status` at here and will update.